libs/dh/metering-point/feature-overview/src/lib/charges/dh-charges.component.html

```diff
@@ -15,7 +15,7 @@
 limitations under the License.
 -->
 <ng-container *rxLet="isLoading$ as isLoading">
-  <div *ngIf="isLoading" class="overview__spinner">
+  <div *ngIf="isLoading" class="charges__spinner">
```

Comment:
Are there any reasons to prefix with charges? Consider removing it as the component properly has isolated styling anyways :)

Reply:
It was prefixed with overview where I hijacked it from but makes sense - I'll remove it :)